src/test/java/com/google/javascript/clutz/ProgramSubject.java

```diff
@@ -26,21 +26,11 @@
  */
 class ProgramSubject extends Subject<ProgramSubject, ProgramSubject.Program> {
 
-  enum Externs {
-    NONE,
-    PLATFORM,
-    THIRD_PARTY_AND_PLATFORM,;
-
-    static Externs fromTestName(String name) {
-      if (name.contains("_3p_externs")) return Externs.THIRD_PARTY_AND_PLATFORM;
-      if (name.contains("_externs")) return PLATFORM;
-      return NONE;
-    }
-  }
-
   /** A stripped down version of Closure's base.js for Clutz tests. */
   private static final SourceFile CLUTZ_GOOG_BASE =
       SourceFile.fromFile("src/test/java/com/google/javascript/clutz/base.js", UTF_8);
+  public boolean withPlatform = false;
+  public String externFile = null;
```

Comment:
went with `extraExternFile` (indeed it is additive on top of platform externs, if the test needs them).